corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_modal.scss

```diff
@@ -57,31 +57,6 @@
     @include absolute-center-modal(20px);
 }
 
-// Modal that has some kind of control (pagination, search box, etc.) in header or footer
-.modal-header.with-controls, .modal-footer.with-controls {
```

Comment:
These are used in [data_corrections_modal.html](https://github.com/dimagi/commcare-hq/blob/d1d9a20e6cb4a808a494b8e24478cd342dda5f99/corehq/apps/reports/templates/reports/partials/data_corrections_modal.html) and [case_property_modal.html](https://github.com/dimagi/commcare-hq/blob/d1d9a20e6cb4a808a494b8e24478cd342dda5f99/corehq/ex-submodules/casexml/apps/case/templates/case/partials/case_property_modal.html).

Reply:
ah i was grepping for the styles WITH the `.`, going to revert

Reply:
reverted